lib/rubocop/cop/style/ternary_parentheses.rb

```diff
@@ -91,7 +91,7 @@ def redundant_parentheses_enabled?
         end
 
         def parenthesized?(node)
-          node.source =~ /^\(.*\)$/
+          node.type == :begin
```

Comment:
I'm new to this whole AST thing but empirically I realized that a braced expression has a `:begin` root node, while non-braced has anything but it... I will change to `begin_type?`